pkgs/development/tools/fsautocomplete/default.nix

```diff
@@ -0,0 +1,52 @@
+{ buildDotnetModule, mkNugetDeps, emptyFile, lib, dotnet-sdk }:
+let
```

Comment:
this doesnt seem to need to be in a `let in` binding, instead have the `buildDotnetModule` at the root of the file

Reply:
@mdarocha this seems to be needed for this line: `<add key="nugetSource" value="${fsautocomplete.passthru.nuget-source}/lib" />`

Reply:
Oh, didn't notice that

Reply:
see the finalAttrs here https://github.com/NixOS/nixpkgs/blob/49966f2c7139521b1c84af4c76c72725e1eafd2e/doc/stdenv/meta.chapter.md#nixos-tests to avoid this